src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityQuantumTank.java

```diff
@@ -450,8 +460,8 @@ public void receiveCustomData(int dataId, PacketBuffer buf) {
             try {
                 this.fluidTank.setFluid(FluidStack.loadFluidStackFromNBT(buf.readCompoundTag()));
             } catch (IOException ignored) {
-                GTLog.logger.warn("Failed to load fluid from NBT in a quantum tank at {} on a routine fluid update",
-                        this.getPos());
+                GTLog.logger.warn("Failed to load fluid from NBT in a quantum tank at " + this.getPos() +
+                        " on a routine fluid update");
             }
```

Comment:
This change does not seem necessary, since both do the same thing.

Reply:
also probably a mistake with rebasing